postgres0.yml

```diff
@@ -46,6 +46,7 @@ postgresql:
     env_dir: /home/postgres/etc/wal-e.d/env
     threshold_megabytes: 10240
     threshold_backup_size_percentage: 30
+  restore: /usr/bin/true
```

Comment:
Good point, actually, the reason I included the full path is due to usage of abspath before passing the command to subprocess.call. But that abspath is completely unnecessary: if the user wants to pass the command from PATH, we shouldn't block it.